substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -149,9 +149,8 @@ pub mod pallet {
 	};
 	use frame_system::{ensure_root, pallet_prelude::*};
 
-	const RANK_COUNT: usize = 9;
-
 	#[pallet::pallet]
+	#[pallet::without_storage_info]
```

Comment:
```suggestion
```
This is not permissible. Storage info must be provided by all new pallets that we write.